ivy_tests/test_ivy/test_frontends/test_paddle/test_random.py

```diff
@@ -7,6 +7,45 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# multinomial
+@handle_frontend_test(
+    fn_tree="paddle.tensor.random.multinomial",
+    input_dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=0.25,
+        max_value=0.25,
+        min_dim_size=4,
+        max_dim_size=4,
+        shape=helpers.get_shape(min_num_dims=1, max_num_dims=2, min_dim_size=2),
+    ),
+    num_samples=st.integers(min_value=1, max_value=2),
```

Comment:
You're not covering the entire space here. For instance, you should expand the x-range beyond just 0.25. Additionally, there's no need to set the min dimensions and maximum dimension size to 4.

Reply:
Actually there is a reason for it I informed Haris as you can see here. https://github.com/unifyai/ivy/pull/20907#discussion_r1306372771 
Basically numpy and jax needs the array so that the sum of all elements always adds up to 1. I think other framework normalize the array themselves. So that is the reason tuned the parameters so that the array is always added upto 1. But it works with other values as well for the other frameworks that normalize it.